src/lib/render-helpers.ts

```diff
@@ -22,26 +22,28 @@ export function classString(
     classInfo: {[name: string]: string|boolean|number}) {
   const o = [];
   for (const name in classInfo) {
-    const v = classInfo[name];
-    if (v) {
+    // we explicitly want a loose truthy check here.
```

Comment:
Maybe state "why" - someone might be wondering a couple of year from now

Reply:
Fixed